feat(content): Bridge HR intro into main narrative and restore supporting features (#8972)
endless-sky/endless-sky: data/wanderer/wanderers start.txt

	to offer
		has "First Contact: Wanderer: offered"
	on offer
		log "People" "Sayari" `Now that the crisis with humanity is over she has volunteered to serve as a translator and be the Hai diplomat to the Wanderers as a capstone for her career.`
		conversation
			`As you are approaching Hai-home, you contact the Hai government and inform them that you have found new aliens to the north. You submit a request for ambassadorial assistance to help you communicate with these creatures, who seem to speak the Hai language. They immediately direct you to a now familiar private government hangar, where a familiar diplomat meets you.`

> 			`	"Good to see you again Captain <last>. I have arranged for some refreshments so please, come and tell me all about these new aliens you have discovered!"`

Reviewer: ```suggestion
			`	"Good to see you again, Captain <last>. I have arranged for some refreshments so please, come and tell me all about these new aliens you have discovered!"`
```
Also please mention that it's Sayari somewhere.

> 			`	"Good to see you again Captain <last>, I have arranged for some refreshments so please, come and tell me all about these new aliens you have discovered!"`

Reviewer 2: ```suggestion
			`	"Good to see you again Captain <last>. I have arranged for some refreshments so please, come and tell me all about these new aliens you have discovered!"`
```

		dialog phrase "generic waypoint on visit"
	on complete
		log "Planted some surveillance devices, designed by the Wanderers, in Unfettered Hai space. A month from now, they will be done collecting data on ship movements in the area, and may provide a hint of where the Alphas who are selling jump drives to the Unfettered are coming from."

> 		log "People" "Hiyamaharu" `Hiyamaharu is an aged Hai ambassador. He appears to have taken of Sayari's role in dealing with humans since her departure from Hai space.`

Reviewer 2: ```suggestion
		log "People" "Hiyamaharu" `Hiyamaharu is an aged Hai ambassador. He appears to have taken up Sayari's role in dealing with humans since her departure from Hai space.`
```

			choice
				`	"Yes."`
					goto jump

> 				`	"You seem to be very familiar with Hai space?"`

Reviewer 3: I'm not sure saying Danforth is "very familiar" with Hai space necessarily fits here. The only thing he's mentioned is that he knows about the Unfettered, and he himself seems to feel that his knowledge might be outdated.

Author: I'm inclined to say that's a problem for the Wanderer's focused PR to deal with (when it happens)

Author: Also, he knows the boundaries of Hai space on a map, which does not appear to have political ownership labelled, and can identify "rebel" systems at a glance from recall. That's a fair degree of familiarity.

On reflection; I'm resolving this as firmly a Wanderer's PR problem if people want to debate the subtleties of that then.

> 				`	"You seem to be very familiar with the Hai's space?"`

Reviewer 4: ```suggestion
				`	"You seem to be very familiar with Hai space?"`
```
Might be fine, I'm just not used to that.

			choice
				`	"Yes."`
					goto jump

> 				`	"You seem to be very familiar with Hai space?"`

Reviewer 2: ```suggestion
				`	"You seem to be very familiar with Hai space."`
```

Reviewer 2: Because this sentence isn't framed in the form of a question, it shouldn't have a question mark at the end.

	on offer
		log "Traveled to the Hai homeworld with Ambassador Sayari to see if she can convince the Hai to take action to restrain the Unfettered from attacking the Wanderers. The most that the Hai were willing to do was to mass a large war fleet near Unfettered space so that the Unfettered have to divert warships there to defend their home worlds."
		conversation

> 			`The Hai's recently elected Council of "Elders" agrees to meet with you and Sayari and the two of you make an effort to avoid directing yourselves towards Teeneep. Your young friend's popularity as an Elder does not extend to inside these walls and, as Sayari quietly informed you beforehand, apparently Teeneep has not explained the nature of her sympathies towards the Unfettered, even to her oldest friends. It would be detrimental to your cause to specifically seek her assent here, on <origin>, for this matter. After you describe the situation with the Wanderers, one of the other elders who seems older and authoritative says, "We had heard rumors of this. The Unfettered have recruited thousands of new warriors by telling stories of planets for us to explore and colonize, and a cowardly and foolish enemy to defeat."`

Author: The first one yes, the second one no.
If the second one is truly needed for US grammar, then we'll need to restructure.

> 			`The Hai's recently elected Council of "Elders" agrees to meet with you and Sayari and the two of you make an effort to avoid directing yourselves towards Teeneep. Your young friend's popularity as an Elder does not extend to inside these walls and, as Sayari quietly informed you beforehand, apparently Teeneep has not explained the nature of her sympathies towards the Unfettered, even to her oldest friends. It would be detrimental to your cause to specifically seek her assent here, on <origin>, for this matter. After you describe the situation with the Wanderers, one of the other elders who seems older and authoritative says, "We had heard rumors of this. The Unfettered have recruited thousands of new warriors by telling stories of planets for us to explore and colonize, and a cowardly and foolish enemy to defeat."`

Reviewer 5: Yea, that's a new clause after that conjunction, breaks down to full 'Teeneep has not explained'.

> 			`The Hai's recently elected Council of "Elders" agrees to meet with you and Sayari and the two of you make an effort to avoid directing yourselves towards Teeneep. Your young friend's popularity as an Elder does not extend to inside these walls and, as Sayari quietly informed you beforehand, apparently Teeneep has not explained the nature of her sympathies towards the Unfettered, even to her oldest friends. It would be detrimental to your cause to specifically seek her assent here, on <origin>, for this matter. After you describe the situation with the Wanderers, one of the other elders who seems older and authoritative says, "We had heard rumors of this. The Unfettered have recruited thousands of new warriors by telling stories of planets for us to explore and colonize, and a cowardly and foolish enemy to defeat."`

Reviewer 5: ```suggestion
			`The Hai's recently elected Council of "Elders" agrees to meet with you and Sayari, and the two of you make an effort to avoid directing yourselves towards Teeneep. As Sayari had quietly informed you beforehand, your young friend's popularity as an Elder does not extend to inside these walls. Apparently, Teeneep has not explained the nature of her sympathies towards the Unfettered, even to her oldest friends. It would be detrimental to your cause to specifically seek her assent here, on <origin>, for this matter. After you describe the situation with the Wanderers, one of the other elders who seems older and authoritative says, "We had heard rumors of this. The Unfettered have recruited thousands of new warriors by telling stories of planets for us to explore and colonize, and a cowardly and foolish enemy to defeat."`
```

What about just this simple switcheroo to separate them?

> 			`The Hai's recently elected Council of "Elders" agrees to meet with you and Sayari, and the two of you make an effort to avoid directing yourselves towards Teeneep. As Sayari had quietly informed you beforehand, your young friend's popularity as an Elder does not extend to inside these walls. Apparently, Teeneep has not explained the nature of her sympathies towards the Unfettered, even to her oldest friends. It would be detrimental to your cause to specifically seek her assent here, on <origin>, for this matter. After you describe the situation with the Wanderers, one of the other elders who seems older and authoritative says, "We had heard rumors of this. The Unfettered have recruited thousands of new warriors by telling stories of planets for us to explore and colonize, and a cowardly and foolish enemy to defeat."`

Author: No 'us' is correct from the perspective that they view themselves and the Hai as one people.
Even from the perspective of the speaker, 'us' is the precise term - because the propaganda is that the opportunities are for "us" (the Hai) to explore, *by* joining the UHai.

> 			`The Hai's recently elected Council of "Elders" agrees to meet with you and Sayari, and the two of you make an effort to avoid directing yourselves towards Teeneep. As Sayari had quietly informed you beforehand, your young friend's popularity as an Elder does not extend to inside these walls. Apparently, Teeneep has not explained the nature of her sympathies towards the Unfettered, even to her oldest friends. It would be detrimental to your cause to specifically seek her assent here, on <origin>, for this matter. After you describe the situation with the Wanderers, one of the other elders who seems older and authoritative says, "We had heard rumors of this. The Unfettered have recruited thousands of new warriors by telling stories of planets for us to explore and colonize, and a cowardly and foolish enemy to defeat."`

Reviewer 6: Yeah, that makes sense. I interpreted that as telling stories of planets for _those new warriors_ to conquer.

			`	Sayari says, "We must not allow the Unfettered to commit genocide in the name of the Hai people."`
			`	Another elder responds, "But, we promised in the Frostmark Accords that we would not interfere in Unfettered space."`
			`	"They have never kept their side of the treaty," says yet another of the elders, "and surely the prevention of genocide is enough reason to break ours."`
			`	Teeneep responds, "You can hardly fault them for that; a treaty under duress is no treaty at all. No amount of begging or negotiating will convince them to stop their war. They recognize only strength, not the pleas of a coward. We must show strength, and so must the Wanderers. We should increase our fleets at Wah Ki, making ourselves more of a threat."`

> 			`	Some of the other Elders nod in agreement, but a smattering of dissent is voiced loudly enough that the older Elder from before interjects, "Perhaps there are other suggestions we may consider?".`

Reviewer 7: ```suggestion
			`	Some of the other Elders nod in agreement, but a smattering of dissent is voiced loudly enough that the older Elder from before interjects, "Perhaps there are other suggestions we may consider?"`
```

				goto threaten
			label attack
			`	"I will not authorize preemptive violence against our own flesh and blood," says one of the elders. "We only defend ourselves against them, never willingly initiate an attack."`
			`	Teeneep and another Elder who seems to be in her corner both chitter the Hai equivalent of "Hear Hear" to this, though she is being very careful with her contributions.`
			label threaten

> 			`	"Perhaps Teeneep is right and it would be enough merely to threaten them," says another of the elders. "We could gather a massive war fleet above Cloudfire. The Unfettered would need to muster a matching fleet above Darkcloak. It would draw many of their warships away from the alien territory."`

Reviewer 7: I've only noticed now that there's probably a lot of places where 'Elder' isn't capitalised. Is a PR necessary for this?

Author: Would like a smidgen of clarity from @W1zrad on the capitalisation of Elders - then I'll just standardise it in this PR.

Reviewer 7: As far as I'm concerned it's a title like Captain or Senator or Member of Parliament. HR usually capitalises 'Elder' while earlier mentions (I guess by MZ) don't.

Author: I will check through for standardising to that in a bit then. :)

				goto threaten
			label attack
			`	"I will not authorize preemptive violence against our own flesh and blood," says one of the elders. "We only defend ourselves against them, never willingly initiate an attack."`
			`	Teeneep and another Elder who seems to be in her corner both chitter the Hai equivalent of "Hear, Hear" to this, though she is being very careful with her contributions.`
			label threaten
			`	"Perhaps Teeneep is right and it would be enough merely to threaten them," says another of the elders. "We could gather a massive war fleet above Cloudfire. The Unfettered would need to muster a matching fleet above Darkcloak. It would draw many of their warships away from the alien territory."`

> 			`	After well over an hour of further discussion, and with calculated backing from Teeneep, they reach the consensus that this is their best option. As you leave the meeting, Sayari exchanges a few words with Teeneep before approaching you, "And now, I would like to visit the Unfettered themselves. They will probably not listen, but perhaps we can talk sense into them. In any event, it is my job to try."`

Reviewer: ```suggestion
			`	After well over an hour of further discussion, and with calculated backing from Teeneep, they reach the consensus that this is their best option. As you leave the meeting, Sayari exchanges a few words with Teeneep before approaching you, "And now, I would like to visit the Unfettered themselves. They will probably not listen, but perhaps we can talk sense into them. In any case, it is my job to try."`
```
Alternatively could do "At any rate".

		log "The Unfettered are unwilling to agree to even a temporary ceasefire with the Wanderers. They seem to think that they will win this war easily."
		event "wanderers: tempest mass production"
		conversation
			`The Unfettered war chiefs gather around General Choot'k to listen to Sayari's plea with condescending smiles on their faces. When she is done speaking, Choot'k says, "This is a war, not a polite dinner party. We will take what we want, when we want it. We are through with waiting on the charity of others."`
			`	"Better to die asserting our strength than to live with our hands outstretched to beg," says one of the chiefs.`
			branch positive
				"reputation: Hai (Unfettered)" > 0

> 			`	You are unable to convince them to change their plans, and as soon as the conversation ends General Choot'k insists that you leave their planet, "You were afforded a safe arrival here because of our history with the mighty Captain <last>, but now you must leave."`

Reviewer 2: ```suggestion
			`	You are unable to convince them to change their plans, and as soon as the conversation ends General Choot'k insists that you leave their planet. "You were afforded a safe arrival here because of our history with the mighty Captain <last>, but now you must leave."`
```

			`	"Beware! Our warriors in orbit are eager to test themselves against this famous human's strength," says one of the chiefs, laughing.`
			`	As you and Sayari board your ship she mutters to you, "I was told to expect this, there's no need to stay and fight."`
			`	You are unable to convince them to change their plans, and as soon as the conversation ends they insist that you leave their planet. "Our warriors in orbit are eager to test this human's strength," says one of the chiefs, laughing.`
				launch
			label positive

> 			`	You are unable to convince them to change their plans, and as soon as the conversation ends General Choot'k insists that you leave their planet, "You are a respected warrior Captain <last>, but now you must leave with this Fettered diplomat."`

Reviewer 2: ```suggestion
			`	You are unable to convince them to change their plans, and as soon as the conversation ends General Choot'k insists that you leave their planet. "You are a respected warrior, Captain <last>, but now you must leave with this Fettered diplomat."`
```

		log "The Unfettered are unwilling to agree to even a temporary ceasefire with the Wanderers. They seem to think that they will win this war easily."
		event "wanderers: tempest mass production"
		conversation
			`The Unfettered war chiefs gather around General Choot'k to listen to Sayari's plea with condescending smiles on their faces. When she is done speaking, Choot'k says, "This is a war, not a polite dinner party. We will take what we want, when we want it. We are through with waiting on the charity of others."`
			`	"Better to die asserting our strength than to live with our hands outstretched to beg," says one of the chiefs.`
			branch positive
				"reputation: Hai (Unfettered)" > 0
			`	You are unable to convince them to change their plans, and as soon as the conversation ends General Choot'k insists that you leave their planet. "You were afforded a safe arrival here because of our history with the mighty Captain <last>, but now you must leave."`
			`	"Beware! Our warriors in orbit are eager to test themselves against this famous human's strength," says one of the chiefs, laughing.`

> 			`	As you and Sayari board your ship she mutters to you, "I was told to expect this, there's no need to stay and fight."`

Reviewer: ```suggestion
			`	As you and Sayari board your ship she mutters to you, "I was told to expect this. There's no need to stay and fight."`
```
